mmv1/products/artifactregistry/Repository.yaml

```diff
@@ -209,15 +215,6 @@ examples:
       # response.
       - 'remote_repository_config.0.disable_upstream_validation'
 parameters:
-properties:
-  - name: 'name'
-    type: String
-    description: |-
-      The name of the repository, for example:
-      "repo1"
-    output: true
-    custom_flatten: 'templates/terraform/custom_flatten/name_from_self_link.tmpl'
-    custom_expand: 'templates/terraform/custom_expand/shortname_to_url.go.tmpl'
   - name: 'repository_id'
```

Comment:
Based on the MMV1 docs, I _think_ `repository_id` and `location` should be parameters, not properties? Previously, everything was under `properties`.